### ISSUE TITLE
Angular Forms touched/untouched

### ISSUE THREAD

**Issue by P-de-Jong** [ionic-team/ionic#12700]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

When typing in an input, the untouched state will change to touched.

**Expected behavior:**

When typing in an input, the untouched state only change to touched on blur.

**Steps to reproduce:**

**Related code:**

**Other information:**

In Angular 4 outside Ionic this works fine...

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages:

    @ionic/cli-utils  : 1.8.1
    ionic (Ionic CLI) : 3.8.1

global packages:

    Cordova CLI : not installed

local packages:

    @ionic/app-scripts : 2.1.3
    Cordova Platforms  : android 6.2.3 windows 5.0.0
    Ionic Framework    : ionic-angular 3.6.0

System:

    Node : v7.10.0
    npm  : 4.2.0
    OS   : Windows 10
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! I am closing this issue because it is a duplicate of Issue #12102. Please track that issue for further updates. Also please comment on that issue with any new information you may have to share.  Thank you for using Ionic

**Comment by RodolfoSilva** [comment]
@kensodemann I sent a PR to fix it #12315
